modules/bitgo/test/v2/unit/coins/utxo/keySignatures.ts

```diff
@@ -0,0 +1,31 @@
+import * as assert from 'assert';
+import { AbstractUtxoCoin } from '@bitgo/abstract-utxo';
+import { Keychain } from '@bitgo/sdk-core';
+
+import { utxoCoins } from './util';
+
+function describeWithCoin(coin: AbstractUtxoCoin) {
+  describe(`verifyKeySignatures for ${coin.getChain()}`, function () {
+    it('should verify key signature of ZEC', async () => {
+      const userKeychain = await coin.keychains().create();
+      const backupKeychain = await coin.keychains().create();
+      const bitgoKeychain = await coin.keychains().create();
```

Comment:
```suggestion
      const [ userKeychain, backupKeychain, bitgoKeychain] = await Promise.all([ 
      coin.keychains().create(),
      coin.keychains().create(),
      coin.keychains().create()
      ]);
```
NIT: we can use promise.all instead of serially creating keychains

Reply:
I don't believe we are making http requests here anyway though

Reply:
These are synchronous operations, Promise.all won't speed them up.